.eslintrc.js

```diff
@@ -277,59 +277,10 @@ module.exports = {
           // which is extremely common and necessary to maintain type safety.
           'a',
           'b',
-          // caught by prevent-abbreviations below, avoid double-flagging
-          'e',
-          'i',
-          'ch',
```

Comment:
Removed these since there would be no longer double-flagging resulting from `prevent-abbreviations` rule